utils/ort/src/main/kotlin/VersionUtils.kt

```diff
@@ -41,6 +41,9 @@ fun filterVersionNames(version: String, names: List<String>, project: String? =
     val fullMatches = names.filter { it.equals(version, ignoreCase = true) }
     if (fullMatches.isNotEmpty()) return fullMatches
 
+    val nearExactMatch = names.filter { it.equals("v$version", ignoreCase = true) }
```

Comment:
I'd prefer this variable to be named more similar to "fullMatches" above, in particular not use both "full" and "exact" terms here. I propose "fullMatchesVPrefix".

Reply:
Alternatively, I'd also be fine with renaming the existing variable to e.g. "exactMatches" and keep using "nearExactMatches" (plural) here.